bin/MadGraph5_aMCatNLO/cards/production/2017/tt0123j_1l_tbar_5f_ckm_LO_MLM/tt0123j_1l_tbar_5f_ckm_LO_MLM_proc_card.dat

```diff
@@ -0,0 +1,14 @@
+import model sm-ckm_no_b_mass
+#switch to diagonal ckm matrix if relevant for speed
+#import model sm-lepton_masses_no_b_mass
```

Comment:
there are still commented commands...

Reply:
Hi Luca, I am not sure why you see comments there. The file is clean: 

https://github.com/bth5032/genproductions/blob/master/bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/tt0123j_1l_tbar_5f_ckm_LO_MLM/tt0123j_1l_tbar_5f_ckm_LO_MLM_proc_card.dat

Reply:
Ahh I see, those are not the cards, I only want to submit the cards in the 13TeV subdirectory.